extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/HttpAuthenticationMechanism.java

```diff
@@ -21,10 +22,16 @@ public interface HttpAuthenticationMechanism {
     Uni<ChallengeData> getChallenge(RoutingContext context);
 
     /**
-     * Returns the required credential types. If there are no identity managers installed that support the
-     * listed types then this mechanism will not be enabled.
+     * If this mechanism delegates authentication to the {@link IdentityProviderManager}, then the mechanism should
+     * signal with which {@link IdentityProvider#getRequestType()} is
```

Comment:
```suggestion
     * provide supported {https://github.com/link AuthenticationType} request types.
```

Reply:
I wasn't sure here about github.com link, but let's comment separately if you can see a space for an improvement please